wiki/configuration.md

```diff
@@ -296,3 +303,13 @@ Eiffel-Intelligence will decrypt password properties with help of the provided s
 Execute Eiffel-Intelligence with "jasypt.encryptor.password=mySecretEncryptorPassword" flag, example:
 
         java -jar eiffel-intelligence-<version>.war --jasypt.encryptor.password=mySecretEncryptorPassword --spring.config.location=/path/to/application.properties
+
+## Removal of unused bindings from RabbitMQ using mongoDB
+
+Eiffel Intelligence stores bindings of a queue in a database with the collection name
+configured with the property **bindingkeys.collection.name**. When start of EI,
+compares the bindingkeys with the property value rabbitmq.binding.key and bindings
+from mongoDB, then removes unused bindings from RabbitMQ and adds new bindings to the
+mongoDB collection. The name of this particular collection is defined by the below property:
```

Comment:
Should we support keeping old/previously defined RabbitMq binding-keys,,, if administrator wants to continue listen on previous and new binding-keys same time?

Then we need a boolean flag that disable this new functionality.

Reply:
Bindingkeys which are not wanted to be used are getting stayed in the queue so removing them and as per requirement adding new binding keys.

Reply:
Yes, thats the Ericsson internal requirements you refer I guess. But this tool open-source and in Eiffel-Community and should be implemented for general use, then we need to consider all use cases.
And if we should make it backward compatible, that is brought by customers/users and Ericsson internal, then old behavior should be handled/work.

Reply:
Actually, as per users choice we are removing the keys,If user requires ex: #binding2, #binding3 (#binding2, #binding3 are present). If #binding2, #binding4 is given (#binding2 remains same, unused #binding3 is removed and new #binding4 is added). Just to understand, You mean it should have a new property, example: bindingKeys.remove: true/false If its true will remove using the new implemented functionality,If false continue with the same functionality?

Reply:
My thought was if users want to have the queue and bindingkeys intact event if EI binding-key configuration change and let the administrator change/remove binding keys.
But we can add that in a new PR, if needed.
So skip my comment for the moment.